### ISSUE TITLE
Finder appears broken

### ISSUE THREAD

**Issue by profjsb** [skyportal/skyportal#875]
Clicking a finding chart, initially gives a warning about a broken websocket connetion, then returns a JSON:

```json
{"status": "error", "message": "'NoneType' object has no attribute 'data'", 
"data": {}, "version": "0.9.dev0"}
```

**Comment by stefanv** [comment]
Pretty sure I introduced that in my requests refactor:

```
    def get_hdu(url):
        response = get_url(url, stream=True, allow_redirects=True)
        if response is None or response.status_code != 200:
            return None

        hdu = fits.open(io.BytesIO(response.content))[0]
        return hdu
```

Because later we do:

```
            hdu = get_hdu(url)
            if np.count_nonzero(hdu.data) > 0:
                hdu.writeto(image_file)
```

**Comment by stefanv** [comment]
Previous version:

```
        response = requests.get(url, stream=True, allow_redirects=True)
        if response.status_code == 200:
            hdu = fits.open(io.BytesIO(response.content))[0]
            return hdu
        else:
             return None
```

So, this fault could happen in either case if the URL request is unsuccessful.

**Comment by profjsb** [comment]
Can you revert/fix as you see fit @stefanv ?

**Comment by stefanv** [comment]
Yes, will do.

**Comment by profjsb** [comment]
Fixed by #888
